tools/npz2scn.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+# npz2scn - tool for extracting semantic scene information from 3dscenegraph
+#           dataset (https://3dscenegraph.stanford.edu/)
+
+import argparse
+import json
+import sys
+
+import numpy as np
+
+
+# Convert ndarrays to python lists so that we can serialize.
+# Tranform coordinates by rotating Y-axis to Z-axis
+def fixCoords(entry):
```

Comment:
If we know `entry` type we can add typing to the func, for example:
from typing import Any, Dict
def fixCoords(entry: Dict[str, Any]) -> None